### ISSUE TITLE
Add More Instructions On The Homepage

### ISSUE THREAD

**Issue by thealphadollar** [metakgp/gyft-serve#3]
Basic information, including but not limited to the following should be added:

- what is an ICS file
- link to how to add ICS to calendar
- how to add ICS to google calendar

**Comment by sahil-shubham** [comment]
Hi, I would like to work on this issue. Is it available? 
We have to add these in the README file, right? because the issue says "On The Homepage".

**Comment by thealphadollar** [comment]
Hey Sahil, thank you for your interest. I'm glad you want to take up this issue.

It would be nice to have these details on README, and we want them on the webpage as well. Webpage is what all the users, non developers, mostly get to see and having instructions there would be really helpful to them. What do you think? Is this a good idea?

**Comment by sahil-shubham** [comment]
I think that it's a good idea, because the very next step after downloading the time-table is to search **how to import events to google calendar**, adding the link to it would save time. In addition to it, we can also add info about .ics files, about metakgp(already mentioned in another issue) etc., for whoever wants to learn more and also add the same to README.

Is there any specific deadline for this issue?

**Comment by thealphadollar** [comment]
That sounds perfect, and I'm glad you came up with an even better idea 🔥

There is no such deadline, please do as you are comfortable and find time. I'm always just a comment away if you need any assistance from me.

**Comment by sahil-shubham** [comment]
okay, thank you. I have already forked and cloned the repository. I have to learn a lot while doing this, but it would be fun. I would let you know if I face any problem.

**Comment by thealphadollar** [comment]
Enjoying is the best part, Sahil!

**Comment by sahil-shubham** [comment]
![build](https://user-images.githubusercontent.com/60931421/91731232-4e6b6580-ebc4-11ea-8848-d761c9743333.png)

I have added the link to import timetable to google calendar. But, I am thinking of making the link active (clickable) only after one has downloaded the ics. This would prevent users from getting confused between downloading the ics file and adding it to the calendar, as they may go to the link before downloading the timetable.

what do you think about this?

**Comment by thealphadollar** [comment]
This looks cool, thanks Sahil! I have a few doubts I hope you won't mind answering.

> I have added the link to import timetable to google calendar. 

What does the button do?

> I am thinking of making the link active (clickable) only after one has downloaded the ics.

Yeah, if we go ahead with adding the button it would be better to avoid confusion as you have rightly pointed out.

**Comment by sahil-shubham** [comment]
> What does the button do?

It opens another tab with the following link (https://calendar.google.com/calendar/r/settings/export?tab=wc) through which they can add the newly downloaded timetable to their calendar.

**Comment by thealphadollar** [comment]
Oh, that is cool. Do you think it would be better to add the link under "Instructions" instead of a button? I'm only skeptical that users will get confused by that and if we provide more instructions it would be better. What do you think?

In any case, we should rename the button to "Add To Google Cal" right?

**Comment by sahil-shubham** [comment]
>Do you think it would be better to add the link under "Instructions" instead of a button?

I think the button would be better than the instructions, as right now even without the intructions each user is downloading the timetable and it is only logical than the next thing he/she would need is a button to add to calendar. Plus, seeing the button disabled at first and then being enabled later would prompt them to click it.  But, I can add instructions next to if you want.

>In any case, we should rename the button to "Add To Google Cal" right?

yes, I have changed it now. This is how it is before clicking the download ics button:-
![disabled](https://user-images.githubusercontent.com/60931421/91751160-eb88c700-ebe1-11ea-85d1-bbac6c1102ff.png)

and after clicking it:-
![enabled](https://user-images.githubusercontent.com/60931421/91751190-fba0a680-ebe1-11ea-9e66-f2eacfd581c7.png)

**Comment by thealphadollar** [comment]
That looks superb!

Yeah, you are right that we should add instructions alongside the button
and keep the button as well. Really appreciate you taking the time to help
me think through.

Do you think it would be best to add instructions on the left of the
buttons, below the image? A few general steps as you see fit ❤️

**Comment by sahil-shubham** [comment]
Instead of adding the instructions next to the button, I have added them on top of the respective buttons. While also colouring the current instruction. The following images can explain better: 
![first-instruction](https://user-images.githubusercontent.com/60931421/91843708-d197c480-ec73-11ea-8f9a-9f0f557a915f.png)
![second-instruction](https://user-images.githubusercontent.com/60931421/91843732-dbb9c300-ec73-11ea-9779-479d83b6e198.png)

I have also added a collapsible info button below, which can be expanded with a click to show the info regarding ics file. Can you also provide with what to write regarding metakgp here, which would also solve #1 
![info](https://user-images.githubusercontent.com/60931421/91843846-0f94e880-ec74-11ea-852a-8db344dd4904.png)

Please let me know what you think and suggest some improvements.

**Comment by thealphadollar** [comment]
Oh! this is beautiful. Thanks a lot.

I just have one doubt, do you think it'll be more useful for users if we replace "Learn more" with "How to use downloaded ICS file"? I think since we have only one question there, it'll be good to have that way, do let me know your thoughts.

> Can you also provide with what to write regarding metakgp here, which would also solve #1

I think we just need one more information, link to this repository. In the existing "Maintained by MetaKGP with <3" we should append "at Github" where Github will be a hyperlink to this repository's URL. The entire message would read "Maintained by MetaKGP with <3 at GitHub". What do you think of this? I believe you can think of better ways if you want as you did with this issue.

**Comment by sahil-shubham** [comment]
>"How to use downloaded ICS file"?

I replaced learn more with this question as I thought a lot of users won't click on it when they just read "Learn more", replacing with this question tells them the direction in which the extra information would be provided. I have also added links to add the ics file to other calendars like outlook and apple calendar.

>"Maintained by MetaKGP with <3 at GitHub"

I liked this a lot, a very subtle way of linking the repo. Though I was trying to find "GitHub" written in a recognizable way, but later settled with a small octocat. 

This is the current look, thank you for spending time to guide me through this : 
![current_look](https://user-images.githubusercontent.com/60931421/91886963-b9da3380-eca7-11ea-8e6d-7126824c1bcb.png)

Let me know if I should create a pull request.

**Comment by thealphadollar** [comment]
The page looks amazing, thank you Sahil for helping us out and
contributing. Please raise a pull request with the changes so that we can
review and make it live ❤️

**Comment by sahil-shubham** [comment]
I have raised the pull request, please review it and suggest the respective changes. This would be my first contribution to an open source project!

**Comment by thealphadollar** [comment]
> This would be my first contribution to an open source project!

I'm glad I could help you with your first contribution - and thanks a lot for the proactiveness in taking up the issue and working on it so diligently. I'm sure you would go on to do many more contributions :sparkles:
